Mathlib/Combinatorics/Quiver/Path.lean

```diff
@@ -31,6 +31,21 @@ inductive Path {V : Type u} [Quiver.{v} V] (a : V) : V → Sort max (u + 1) v
   | cons : ∀ {b c : V}, Path a b → (b ⟶ c) → Path a c
 #align quiver.path Quiver.Path
 
+/-- A computable version of `Path.rec`. Workaround until Lean has native support for this. -/
```

Comment:
Can you reference the lean4 issue in a comment above?

Reply:
also,
```suggestion
/-- A computable version of `Quiver.Path.rec`. Workaround until Lean has native support for this. -/
```

Reply:
Works like I just did?